### PR TITLE
Extended tracevis.py

### PR DESCRIPTION
**Pull request by Xeratec**
I had problems to visualize large traces (>1 GB) generated with Banshee with the `about:tracing` functionality in Google Chrome. To tackle this problem, I extended the `tracevis.py` script to support the new [Perfetto UI](https://ui.perfetto.dev/) and added the option to compress the traces by only showing the function name instead of every single instruction.

Additionally, I implemented the option to information captured sections between `mempool_start_benchmark()` and `mempool_stop_benchmark()` only. This is particularly relevant for Banshee traces.

Backward compatibility with the about:tracing feature in Google Chrome is still maintained.

### Added
- Support the new [Perfetto UI](https://ui.perfetto.dev/)
- Support compression of large traces by extracting the function and only display them
- Support filtering calls between  `mempool_start_benchmark()` and `mempool_stop_benchmark()` calls

## Checklist

- [x] Automated tests pass
- [x] Changelog updated
- [x] Code style guideline is observed

## Appendix
**Google Chrome Trace-Viewer**
JSON Trace Size: 44 MB
image

**Perfetto UI (Enabled filtering)**
JSON Trace Size: 39 MB
image

**Perfetto UI (Enabled functions extraction and filtering)**
JSON Trace Size: 52 kB
image

### CODE REVIEW

File: scripts/tracevis.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
 
-# Copyright 2021 ETH Zurich and University of Bologna.
+# Copyright 2023 ETH Zurich and University of Bologna.
```
Comment (SamuelRiedel) [inline scripts/tracevis.py:3]: Can you please revert this change? The date in copyright headers should reflect the date the file was first published.

**Review by SamuelRiedel — CHANGES_REQUESTED** [review]
Thanks a lot. That's a very nice enhancement, and everything looks good to me. I just had one minor nitpick.

File: scripts/tracevis.py (hunk 1 above)
Comment (Xeratec) [reply scripts/tracevis.py]: Oh, yes sure. Good to know

**Review by SamuelRiedel — APPROVED** [review]
